src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -4039,113 +4039,83 @@ namespace ZoneEquipmentManager {
-            // init each sequenced demand to the full output
-            if (allocated(ZoneSysMoistureDemand(ZoneNum).SequencedOutputRequired))
-                ZoneSysMoistureDemand(ZoneNum).SequencedOutputRequired = ZoneSysMoistureDemand(ZoneNum).TotalOutputRequired; // array assignment
-            if (allocated(ZoneSysMoistureDemand(ZoneNum).SequencedOutputRequiredToHumidSP))
-                ZoneSysMoistureDemand(ZoneNum).SequencedOutputRequiredToHumidSP =
-                    ZoneSysMoistureDemand(ZoneNum).OutputRequiredToHumidifyingSP; // array assignment
-            if (allocated(ZoneSysMoistureDemand(ZoneNum).SequencedOutputRequiredToDehumidSP))
-                ZoneSysMoistureDemand(ZoneNum).SequencedOutputRequiredToDehumidSP =
-                    ZoneSysMoistureDemand(ZoneNum).OutputRequiredToDehumidifyingSP; // array assignment
-        } else if (FirstHVACIteration && (ZoneEquipList(ZoneNum).LoadDistScheme != DataZoneEquipment::LoadDist::SequentialLoading) &&
-                   (ZoneEquipList(ZoneNum).LoadDistScheme != DataZoneEquipment::LoadDist::UniformLoading)) {
-            // init each sequenced demand to the zone design load in order to get available capacities from equipment
-            if (allocated(ZoneSysEnergyDemand(ZoneNum).SequencedOutputRequired)) {
-                if (ZoneSysEnergyDemand(ZoneNum).TotalOutputRequired >= 0.0) {
-                    ZoneSysEnergyDemand(ZoneNum).SequencedOutputRequired = DataSizing::FinalZoneSizing(ZoneNum).DesHeatLoad; // array assignment
-                } else {
-                    ZoneSysEnergyDemand(ZoneNum).SequencedOutputRequired = -DataSizing::FinalZoneSizing(ZoneNum).DesCoolLoad; // array assignment
-                }
-            }
-            if (allocated(ZoneSysEnergyDemand(ZoneNum).SequencedOutputRequiredToHeatingSP)) {
-                if (ZoneSysEnergyDemand(ZoneNum).TotalOutputRequired >= 0.0) {
-                    ZoneSysEnergyDemand(ZoneNum).SequencedOutputRequiredToHeatingSP =
-                        DataSizing::FinalZoneSizing(ZoneNum).DesHeatLoad; // array assignment
-                } else {
-                    ZoneSysEnergyDemand(ZoneNum).SequencedOutputRequiredToHeatingSP =
-                        -DataSizing::FinalZoneSizing(ZoneNum).DesCoolLoad; // array assignment
-                }
-            }
-            if (allocated(ZoneSysEnergyDemand(ZoneNum).SequencedOutputRequiredToCoolingSP)) {
-                if (ZoneSysEnergyDemand(ZoneNum).TotalOutputRequired >= 0.0) {
-                    ZoneSysEnergyDemand(ZoneNum).SequencedOutputRequiredToCoolingSP =
-                        DataSizing::FinalZoneSizing(ZoneNum).DesHeatLoad; // array assignment
-                } else {
-                    ZoneSysEnergyDemand(ZoneNum).SequencedOutputRequiredToCoolingSP =
-                        -DataSizing::FinalZoneSizing(ZoneNum).DesCoolLoad; // array assignment
+        energy.RemainingOutputRequired = energy.TotalOutputRequired;
```

Comment:
I like the change to use aliases (const refs), though in this case it does muddy that change with the actual functional changes you made.  In a super-ideal world, we would do a clean-up PR first that made no functional changes, and get that in, followed up by the actual functional changes.  But the world isn't quite super-ideal I guess :upside_down_face: .

Reply:
Valid point. I can revert these if that would help.